src/main/kotlin/ar/edu/unahur/obj2/vendedores/CentroDeDistribucion.kt

```diff
@@ -0,0 +1,20 @@
+package ar.edu.unahur.obj2.vendedores
+
+class CentroDeDistribucion(ciudad: Ciudad,private var vendedores: MutableList<Vendedor>) {
```

Comment:
Otra forma es inicializar la lista directamente en la clase, y no aceptar un parámetro en el constructor:

```suggestion
class CentroDeDistribucion(ciudad: Ciudad) {
  val vendedores = mutableListOf<Vendedor>()
```